src/app/hooks/usePWAInstallTracker/index.tsx

```diff
@@ -0,0 +1,28 @@
+import { useEffect, useRef } from 'react';
+import useCustomEventTracker from '../useCustomEventTracker';
+
+/**
+ * A hook to track PWA installation events using Reverb Analytics
+ */
+const usePWAInstallTracker = () => {
+  const { trackEvent } = useCustomEventTracker({ eventName: 'pwa-installed' });
+
+  const hasTracked = useRef(false);
+
+  useEffect(() => {
+    function handleAppInstalled() {
```

Comment:
Can this be a fat arrow function? 
```suggestion
    const handleAppInstalled = () => {
```

Reply:
Updated